$OpenBSD: patch-nss_lib_freebl_blapi_h,v 1.3 2016/03/04 16:29:24 landry Exp $

From firefox in pkgsrc:
SHA1_Update conflicts with openssl which may be dynamically loaded

--- nss/lib/freebl/blapi.h.orig	Fri Feb 26 21:51:11 2016
+++ nss/lib/freebl/blapi.h	Thu Mar  3 11:01:55 2016
@@ -1218,6 +1218,8 @@ extern void SHA1_DestroyContext(SHA1Context *cx, PRBoo
 */
 extern void SHA1_Begin(SHA1Context *cx);
 
+#define SHA1_Update NSS_SHA1_Update
+
 /*
 ** Update the SHA-1 hash function with more data.
 **	"cx" the context
